### ISSUE TITLE
Ctrl+Alt+` no longer works

### ISSUE THREAD

**Issue by rmwiseman** [linuxmint/cinnamon#7776]
```
 * Cinnamon version - Cinnamon 3.8.8

 * Distribution - Mint 19
 * Graphics hardware *and* driver used - Haswell-ULT Integrated Graphics Controller
 * 64 bit
 ```

**Issue**

In Mint 18, I could use Alt+\` to switch between windows in the same application and on the same workspace.  This still works in Mint 19.

In Mint 18, I could also use Ctrl+Alt+\` to switch between windows in the same application across all workspaces.  In Mint 19, this no longer works.

**Steps to reproduce**

Open windows for an application on multiple workspaces and use Ctrl+Alt+` to switch between its windows; nothing happens.

**Expected behaviour**

Same behaviour as Alt+\` but with windows from all workspaces.

**Other information**

**Comment by rmwiseman** [comment]
I'm also using 3.8.9 but Alt+` definitely doesn't switch between windows from all workspaces for me.

**Comment by icarter09** [comment]
@rmwiseman are you still experiencing this issue?

**Comment by rmwiseman** [comment]
> @rmwiseman are you still experiencing this issue?

Yes, for me, Alt+` still switches between windows of the same application on the same workspace, but Ctrl+Alt+` does nothing.

I'm now on Mint 19.1 and Cinnamon 4.0.10.

**Comment by smurphos** [comment]
Hi - a related issue - https://github.com/linuxmint/muffin/issues/352

**Comment by rmwiseman** [comment]
Still an issue in Mint 21.1.

**Comment by mtwebster** [comment]
I just tested Mint 18, 19, 20 and 21 and they all share the same keybinding sets (at least as far as switching goes).

There's:
- alt-tab, shift-alt-tab - switch all windows on current workspace
- ctrl-alt-tab, shift-ctrl-alt-tab - switch all windows on all workspaces
- alt-~, shift-alt-~ - switch all same app on current workspace.

You can enable this using the normal alt-` by opening Window Settings -> Alt-Tab -> Show windows from all workspaces.

**Comment by rmwiseman** [comment]
> You can enable this using the normal alt-` by opening Window Settings -> Alt-Tab -> Show windows from all workspaces.

Thanks @mtwebster, that really ought to be made clearer, as I (foolishly?) assumed that the Alt-Tab options were only for Alt-Tab!

**Comment by rmwiseman** [comment]
Tried today on Mint 22.3 (Cinnamon 6.6.7) and it now behaves as it used to — even without the "show windows from all workspaces" enabled.
